templates/config.json

```diff
@@ -537,6 +558,10 @@
       "provider": "",
       "summary": "The organization incorporates best practices such as least-privilege or zero-trust in its security operating model.",
       "guidance": "This describes the high level operating principles of a DevOps oriented security program. The cloud-native, zero-trust approach enables a modern security operating model and eliminates the need for certain legacy security controls.",
+      "instructionsSummary": "Confirm/tailor HIPAA/HITECH relevance",
+      "instructions": [
+        "Are you HIPAA/HITECH compliant?"
```

Comment:
I don't see the relevance/reference to HIPAA/HITECH compliance for this specific procedure. I'd specifically highlight confirming:
-zero trust
-microservice architecture
-money back guarantee/reimburse for their losses
-reference to Chef/AWS specific tooling
-monthly security training

Reply:
I agree with @tanggk